src/libstore/include/nix/store/uds-remote-store.hh

```diff
@@ -68,7 +68,7 @@ struct UDSRemoteStore : virtual IndirectRootStore, virtual RemoteStore
 
     void narFromPath(const StorePath & path, Sink & sink) override
     {
-        Store::narFromPath(path, sink);
+        RemoteStore::narFromPath(path, sink);
```

Comment:
I don't think this should change, because this is client-side. That to me is an orthogonal thing and the domain of https://github.com/NixOS/nix/pull/9429 per the above.